runtime/src/append_vec.rs

```diff
@@ -96,6 +98,8 @@ impl AppendVec {
             }
         }
 
+        AppendVec::sanitize_mmap_size(size).unwrap();
```

Comment:
yes, `unwrap()`ing here just because considering this is under the supertubes milestone and there isn't enough time to do large refacotoring. There are many other security issues to be fixed for snapshot for the milesone.

I'm pretty sure this code (=AppendVec::new) isn't touched by the snapshot deserialization code path at all.

Reply:
Move this above `if create {`